src/task/methods/scape/script.py

```diff
@@ -76,8 +97,9 @@
 for i, d in enumerate(drugs):
 	print(i, d)
 	scm = scape.model.create_default_model(par["n_genes"], df_de, df_lfc)
+	cell = confirm_celltype(par["cell"], d)
```

Comment:
if the other suggestion is accepted

```suggestion
	cell = confirm_celltype(df_de, par["cell"], d)